generate_PSC.py

```diff
@@ -4,6 +4,7 @@
 from astropy.io import fits
 import astropy.units as u
 from astropy.coordinates import SkyCoord
+import json
```

Comment:
Are you using a style guide?

This isn't something we've discussed on this project, but we might as well start going with one.  This one seems popular https://www.python.org/dev/peps/pep-0008/